nixos/modules/services/video/go2rtc/default.nix

```
@@ -55,8 +55,8 @@ in
           ffmpeg = {
             bin = mkOption {
               type = path;
-              default = "${lib.getBin pkgs.ffmpeg_6-headless}/bin/ffmpeg";
-              defaultText = literalExpression "\${lib.getBin pkgs.ffmpeg_6-headless}/bin/ffmpeg";
+              default = lib.getExe pkgs.ffmpeg_7-headless;
+              defaultText = literalExpression "lib.getExe pkgs.ffmpeg_7-headless";
```

```suggestion
              default = lib.getExe pkgs.ffmpeg-headless;
              defaultText = literalExpression "lib.getExe pkgs.ffmpeg-headless";
```

Feel free to unpin entirely. Upstream's ffmpeg support story is good.

---

I plan to optimistically unpin all FFmpeg 7 dependencies in the 6 → 7 bump with the intention of more aggressively pinning only those packages that truly need it going forward. So as with my previous PRs moving off old FFmpeg versions, this is just executable documentation that it does indeed work with 7 while the default is still 6 for now. I can change it if you’d like, but I’d prefer to get FFmpeg 7 deployed here now while waiting for the 6 → 7 bump in the next staging cycle.

---

Not enough fucks given.